heat/sparse/tests/test_dcsrmatrix.py

```diff
@@ -0,0 +1,287 @@
+import unittest
+import heat as ht
+import torch
+
+from heat.core.tests.test_suites.basic_test import TestCase
+
+from typing import Tuple
+
+
+@unittest.skipIf(
+    int(torch.__version__.split(".")[1]) < 10,
+    f"ht.sparse requires torch >= 1.10. Found version {torch.__version__}.",
+)
```

Comment:
@Mystic-Slice please include the major version in this check (and all subsequent), torch will upgrade to 2.0 soon. Thanks!

Reply:
Done!